publish/publish.proj

```diff
@@ -99,6 +99,22 @@
                    FinalizeContainer="Runtime/$(SharedFrameworkNugetVersion)"
                    ForcePublish="true"                  
                    Condition="'@(_MissingBlobNames)' == ''" />
+    <Error Condition="'$(ChecksumAzureAccessToken)' == ''" Text="Missing required property 'ChecksumAzureAccessToken'" />
+    <Error Condition="'$(ChecksumAzureAccountName)' == ''" Text="Missing required property 'ChecksumAzureAccountName'" />
+    <Error Condition="'$(ChecksumContainerName)' == ''" Text="Missing required property 'ChecksumContainerName'" />
+    <Message Importance="High" Text="Finalizing Checksums" />
+    <FinalizeBuild AccountName="$(ChecksumAzureAccountName)"
```

Comment:
Ah, ok.  I get it.

As long as the FinalizeBuild task doesn't do something that should only be done for the real assets account, then this approach should work.